tests/integration-tests/tests/common/schedulers_common.py

```diff
@@ -469,6 +469,12 @@ def get_node_info(self, nodename):
         """Get node info."""
         return self._remote_command_executor.run_remote_command("scontrol show nodes {0}".format(nodename))
 
+    def get_conf_param(self, param):
+        """Get value of configuration parameter."""
+        result = self._remote_command_executor.run_remote_command("scontrol show config | grep {0}".format(param))
+        match = re.search(r"(\s+)= (.*)$", result.stdout)
```

Comment:
Are we sure that we only need to parse setting values that do not require any escaping, e.g. `param_name = 'foo bar foo'`?